solidity/contracts/BurntPixArchives.sol

```diff
@@ -50,7 +50,7 @@ struct Contribution {
 // Registry implements the NFT existence and ownership tracking.
 contract BurntPixArchives is LSP8CappedSupply {
     address public immutable fractalClone;
-    address public immutable archiveHelpers;
+    address public immutable archiveHelpers; //do we need a setter for this in case we want to fix some logic later? but I guess it'd be less trustworthy
```

Comment:
yeah. imo we should aim for immutability. at the end of the day this is an experiment with no expectations. we should aim to do our best to prevent any issues but once it's live it's live. the only replacement can be a whole new experiment. @democrazyfi